README.md

```diff
@@ -35,7 +35,7 @@ description: example
 
 * `version` is a **required** parameter that indicates the configure file's version, the default version is `v2rc1`.
```

Comment:
`version` is a **required** parameter that indicates the version number of the configuration file. The default version is `v2rc1`.